src/en/credentials.md

```diff
@@ -46,8 +60,8 @@ have more than one for a cloud:
 juju set-default-credential <cloud> <credential>
 ```
 
-If you are having difficulty finding the correct credential information for 
-your cloud, please see the [help pages][help].
+The default credentials do not need to be specified on the commandline for 
+bootstrap (creating a controller).  
```

Comment:
I was trying to clarify why there were such things as default credentials. 
The credentials do not need to be specified for bootstrap if you have a default. Of course you don't need to specify a default if you are going to specify the credential with the actual command. I will reword it then.